addons/stock/models/stock_move_line.py

```diff
@@ -847,6 +847,8 @@ def _get_aggregated_properties(self, move_line=False, move=False):
         elif description.startswith(product.name):
             description = description.removeprefix(product.name).strip()
         line_key = f'{product.id}_{product.display_name}_{description or ""}_{uom.id}'
+        if uom != packaging_uom:
+            line_key += f'_{packaging_uom.id}'
```

Comment:
You can always add `packaging_uom` to the key. We don't have to check if they're different to add it to the key.
```suggestion
        line_key = f'{product.id}_{product.display_name}_{description or ""}_{uom.id}_{packaging_uom.id}'
```

Reply:
@hapt-odoo Sorry, I didn't notice that at first, but now you have to modify some tests that use this key.